.github/ISSUE_TEMPLATE/bug.yml

```diff
@@ -1,6 +1,5 @@
 name: "Bug report"
```

Comment:
ultra-mega-important change required:
add bug emoji for project wide consistency